src/main/java/io/stargate/sgv2/jsonapi/exception/JsonApiException.java

```diff
@@ -58,8 +60,17 @@ public CommandResult get() {
   }
 
   public CommandResult.Error getCommandResultError(String message) {
-    Map<String, Object> fields =
-        Map.of("errorCode", errorCode.name(), "exceptionClass", this.getClass().getSimpleName());
+    Map<String, Object> fields = null;
+    SmallRyeConfig config = ConfigProvider.getConfig().unwrap(SmallRyeConfig.class);
+    // enable debug mode for unit tests, since it can not be injected
```

Comment:
I'm not sure I understand this... is there something special wrt unit tests?
Or do you mean that we cannot use injection here and need to explicitly access configuration this way?